src/module.ts

```diff
@@ -354,7 +355,16 @@ export default defineNuxtModule<NuxtI18nOptions>({
     ])
 
     /**
-     * transpile @nuxtjs/i18n
+     * Directive transformation for `v-t` SSR
+     */
+
+    // Use `legacy` as global injections are prefixed (`$t`)
+    const transformVT = transformVTDirective({ mode: 'legacy' })
```

Comment:
I think `mode` option should be set from [`i18n.types`](https://i18n.nuxtjs.org/docs/options/misc#types)

Reply:
Maybe I'm misunderstanding, but the injected component properties are always prefixed with `$` (such as `$t()`) right? 

When passing `mode: 'composition'` to the transform it will (in this case incorrectly) transform the code to use `t()` without prefix, which I think won't work 🤔.